environment.yml

```diff
@@ -0,0 +1,17 @@
+name: chemprop
+channels:
+  - pytorch
+  - conda-forge
+dependencies:
+  - python=3.11
+  - pytorch::pytorch
```

Comment:
our `pyproject.toml` specifies `torch >= 2.1` [here](https://github.com/chemprop/chemprop/blob/ec55383d2888ae0a2d631d76df28c5fef3b7536c/pyproject.toml#L36). Can we specify that in this line? I'm not familiar with the `::` syntax, but maybe you can still just append `>=2.1` to the end of this line?

Reply:
yes, that should be fine!